types/index.d.ts

```diff
@@ -0,0 +1,49 @@
+// TypeScript Version: 3.7
+
+import {VFile, VFileCompatible, VFileOptions} from 'vfile'
+
+export = rename
+
+/**
+ * When given something, returns a vfile from that, and changes its path properties.
+ * - If there is no bound rename (it’s null or undefined), makes sure file is a VFile
+ * - If the bound rename is a normal string starting with a dot (.), sets file.extname
+ * - Otherwise, if the bound rename is a normal string, sets file.basename
+ * - If the bound test is an array, all renames in it are performed
+ * - Otherwise, if the bound rename is an object, renames according to the Spec
+ * @param file VFile to rename
+ * @returns The renamed `file`
+ */
+declare function move(file: VFile): VFile
+
+/**
+ * Renames the given `file` with `renames`
+ * @param file VFile to rename
+ * @param renames Rename instructions
+ * @returns The renamed `file`
+ */
+declare function rename(file?: VFileCompatible, renames?: rename.Renames): VFile
+
+declare namespace rename {
+  interface SpecAffix {
+    prefix?: string
+    suffix?: string
+  }
+
+  interface Spec {
```

Comment:
```suggestion
  /**
   * A spec is an object describing path properties to values. For each property in spec, if its value is string, the value of the path property on the given file is set. If the value is object, it can have a prefix or suffix key, the value of the path property on the given file is prefixed and/or suffixed.
   */
  interface Spec {
```